diff --git a/CMakeLists.txt b/CMakeLists.txt
index b107007..134024f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,7 +39,7 @@ if(NOT DEFINED KDEPIM_RUNTIME_DEV_VERSION)
     set(KDEPIM_RUNTIME_DEV_VERSION "")
 endif()
 
-set(KDEPIM_RUNTIME_VERSION_NUMBER "5.5.1")
+set(KDEPIM_RUNTIME_VERSION_NUMBER "5.5.2")
 set(KDEPIM_RUNTIME_VERSION "${KDEPIM_RUNTIME_VERSION_NUMBER}${KDEPIM_RUNTIME_DEV_VERSION}")
 
 configure_file(kdepim-runtime-version.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/kdepim-runtime-version.h @ONLY)
@@ -63,25 +63,25 @@ set(QT_REQUIRED_VERSION "5.6.0")
 
 set(KDEPIMRUNTIME_LIB_VERSION "${KDEPIM_RUNTIME_VERSION_NUMBER}")
 set(KDEPIMRUNTIME_LIB_SOVERSION "5")
-set(AKONADI_VERSION "5.5.1")
-
-set(KCONTACTS_LIB_VERSION "5.5.1")
-set(KCALENDARCORE_LIB_VERSION "5.5.1")
-set(IDENTITYMANAGEMENT_LIB_VERSION "5.5.1")
-set(KMAILTRANSPORT_LIB_VERSION "5.5.1")
-set(CALENDARUTILS_LIB_VERSION "5.5.1")
-set(KDAV_LIB_VERSION "5.5.1")
-set(KIMAP_LIB_VERSION "5.5.1")
-set(KMBOX_LIB_VERSION "5.5.1")
-set(AKONADICALENDAR_LIB_VERSION "5.5.1")
-set(KONTACTINTERFACE_LIB_VERSION "5.5.1")
-set(AKONADIKALARM_LIB_VERSION "5.5.1")
-set(KMIME_LIB_VERSION "5.5.1")
-set(XMLRPCCLIENT_LIB_VERSION "5.5.1")
-set(KCONTACTS_LIB_VERSION "5.5.1")
-set(AKONADIMIME_LIB_VERSION "5.5.1")
-set(AKONADICONTACT_LIB_VERSION "5.5.1")
-set(AKONADINOTE_LIB_VERSION "5.5.1")
+set(AKONADI_VERSION "5.5.2")
+
+set(KCONTACTS_LIB_VERSION "5.5.2")
+set(KCALENDARCORE_LIB_VERSION "5.5.2")
+set(IDENTITYMANAGEMENT_LIB_VERSION "5.5.2")
+set(KMAILTRANSPORT_LIB_VERSION "5.5.2")
+set(CALENDARUTILS_LIB_VERSION "5.5.2")
+set(KDAV_LIB_VERSION "5.5.2")
+set(KIMAP_LIB_VERSION "5.5.2")
+set(KMBOX_LIB_VERSION "5.5.2")
+set(AKONADICALENDAR_LIB_VERSION "5.5.2")
+set(KONTACTINTERFACE_LIB_VERSION "5.5.2")
+set(AKONADIKALARM_LIB_VERSION "5.5.2")
+set(KMIME_LIB_VERSION "5.5.2")
+set(XMLRPCCLIENT_LIB_VERSION "5.5.2")
+set(KCONTACTS_LIB_VERSION "5.5.2")
+set(AKONADIMIME_LIB_VERSION "5.5.2")
+set(AKONADICONTACT_LIB_VERSION "5.5.2")
+set(AKONADINOTE_LIB_VERSION "5.5.2")
 
 
 set( SHARED_MIME_INFO_MINIMUM_VERSION "0.40" )
@@ -140,7 +140,7 @@ set_package_properties(Sasl2 PROPERTIES
 )
 
 # Extra package
-find_package(KPimGAPI "5.5.1" CONFIG)
+find_package(KPimGAPI "5.5.2" CONFIG)
 
 
 add_subdirectory(resources)
diff --git a/resources/dav/services/owncloud-pre9.desktop b/resources/dav/services/owncloud-pre9.desktop
index 0bf50b8..8d2a425 100644
--- a/resources/dav/services/owncloud-pre9.desktop
+++ b/resources/dav/services/owncloud-pre9.desktop
@@ -10,6 +10,7 @@ Name[en_GB]=ownCloud (> 5.0 and < 9.0)
 Name[es]=ownCloud (> 5.0 y < 9.0)
 Name[et]=ownCloud (> 5.0 ja < 9.0)
 Name[fi]=ownCloud (> 5.0 ja < 9.0)
+Name[fr]=ownCloud (> 5.0 et < 9.0)
 Name[gl]=ownCloud (> 5.0 e < 9.0)
 Name[it]=ownCloud (> 5.0 e < 9.0)
 Name[ko]=ownCloud (> 5.0 및 < 9.0)
diff --git a/resources/ical/icalresource.desktop b/resources/ical/icalresource.desktop
index 1e0db19..e7be6a4 100644
--- a/resources/ical/icalresource.desktop
+++ b/resources/ical/icalresource.desktop
@@ -6,6 +6,7 @@ Name[cs]=Soubor s kalendářem iCal
 Name[de]=ICal-Kalenderdatei
 Name[en_GB]=iCal Calendar File
 Name[es]=Archivo de calendario iCal
+Name[fi]=iCal-kalenteritiedosto
 Name[it]=File di calendario iCal
 Name[ko]=iCal 달력 파일
 Name[nl]=iCal-agendabestand
diff --git a/resources/ical/wizard/icalwizard.desktop b/resources/ical/wizard/icalwizard.desktop
index 6a10ade..31d354b 100644
--- a/resources/ical/wizard/icalwizard.desktop
+++ b/resources/ical/wizard/icalwizard.desktop
@@ -6,6 +6,7 @@ Name[cs]=Soubor s kalendářem iCal
 Name[de]=ICal-Kalenderdatei
 Name[en_GB]=iCal Calendar File
 Name[es]=Archivo de calendario iCal
+Name[fi]=iCal-kalenteritiedosto
 Name[it]=File di calendario iCal
 Name[ko]=iCal 달력 파일
 Name[nl]=iCal-agendabestand
diff --git a/resources/icaldir/icaldirresource.desktop b/resources/icaldir/icaldirresource.desktop
index 6e98296..f06cff8 100644
--- a/resources/icaldir/icaldirresource.desktop
+++ b/resources/icaldir/icaldirresource.desktop
@@ -6,6 +6,7 @@ Name[cs]=Složka kalendáře iCal
 Name[de]=ICal-Kalenderordner
 Name[en_GB]=iCal Calendar Folder
 Name[es]=Carpeta de calendario iCal
+Name[fi]=iCal-kalenterikansio
 Name[it]=Cartella di calendario iCal
 Name[ko]=iCal 달력 폴더
 Name[nl]=iCal-agendamap
diff --git a/resources/imap/imapresource.kcfg b/resources/imap/imapresource.kcfg
index b003e95..5f69205 100644
--- a/resources/imap/imapresource.kcfg
+++ b/resources/imap/imapresource.kcfg
@@ -38,7 +38,7 @@
   <group name="cache">
     <entry name="DisconnectedModeEnabled" type="Bool">
       <label>Defines if all the IMAP data should be cached locally all the time</label>
-      <default>false</default>
+      <default>true</default>
     </entry>
     <entry name="IntervalCheckEnabled" type="Bool">
        <label>Defines if interval checking is enabled.</label>
diff --git a/resources/imap/setupserver.cpp b/resources/imap/setupserver.cpp
index 12a37ae..2912972 100644
--- a/resources/imap/setupserver.cpp
+++ b/resources/imap/setupserver.cpp
@@ -34,6 +34,7 @@
 #include "serverinfodialog.h"
 #include "folderarchivesettingpage.h"
 #include "utils.h"
+#include "config.h"
 
 #include <mailtransport/transport.h>
 #include <mailtransport/servertest.h>
@@ -596,8 +597,8 @@ void SetupServer::slotComplete()
 
 void SetupServer::slotSafetyChanged()
 {
-#ifdef WITH_GMAIL_XOAUTH2
     if (m_serverTest == nullptr) {
+#ifdef WITH_GMAIL_XOAUTH2
         const bool isGmail = Utils::isGmail(m_ui->imapServer->text());
         qCDebug(IMAPRESOURCE_LOG) << "serverTest null";
         m_ui->noRadio->setEnabled(!isGmail);
@@ -605,9 +606,9 @@ void SetupServer::slotSafetyChanged()
         m_ui->tlsRadio->setEnabled(!isGmail);
 
         m_ui->authenticationCombo->setEnabled(!isGmail);
+#endif
         return;
     }
-#endif
     QVector<int> protocols;
 
     switch (m_ui->safeImapGroup->checkedId()) {
diff --git a/resources/tomboynotes/tomboynotesresource.desktop b/resources/tomboynotes/tomboynotesresource.desktop
index 60f98e3..f7576e8 100644
--- a/resources/tomboynotes/tomboynotesresource.desktop
+++ b/resources/tomboynotes/tomboynotesresource.desktop
@@ -9,6 +9,7 @@ Name[en_GB]=Akonadi TomboyNotes Resource
 Name[es]=Recurso de notas de Tomboy de Akonadi
 Name[et]=Akonadi Tomboy sedelite ressurss
 Name[fi]=Akonadin TomboyNotes-resurssi
+Name[fr]=Ressource Akonadi TomboyNotes
 Name[gl]=Recurso de notas de Tomboy para Akonadi
 Name[it]=Risorsa note di Tomboy per Akonadi
 Name[ko]=Akonadi TomboyNotes 자원
@@ -35,6 +36,7 @@ Comment[en_GB]=Resource for Tomboy compatible notes server
 Comment[es]=Recurso para un servidor de notas compatible con Tomboy
 Comment[et]=Tomboyga ühilduvate sedelite serveri ressurss
 Comment[fi]=Tomboy-yhteensopivan muistiinpanopalvelimen resurssi
+Comment[fr]=Ressource pour serveur de notes compatible avec Tomboy
 Comment[gl]=Recurso para servidores de notas compatíbeis con Tomboy.
 Comment[it]=Risorsa per un server di note compatibile con Tomboy
 Comment[ko]=Tomboy 호환 메모 서버 자원
diff --git a/resources/vcarddir/wizard/vcarddirwizard.desktop b/resources/vcarddir/wizard/vcarddirwizard.desktop
index e55a701..95636d1 100644
--- a/resources/vcarddir/wizard/vcarddirwizard.desktop
+++ b/resources/vcarddir/wizard/vcarddirwizard.desktop
@@ -51,6 +51,7 @@ Comment[en_GB]=Loads contact from vCard Directory
 Comment[es]=Carga contactos desde un directorio vCard
 Comment[et]=Kontakti laadimine vCardide kataloogist
 Comment[fi]=Lataa yhteystiedon VCard-kansiosta
+Comment[fr]=Charge des données depuis un dossier « vCard »
 Comment[gl]=Carga contactos dun directorio vCard.
 Comment[it]=Carica contatti da una cartella vCard
 Comment[ko]=vCard 디렉터리에서 연락처를 가져옵니다
