--- a/migration/kmail/kmailmigrator.cpp
+++ b/migration/kmail/kmailmigrator.cpp
@@ -46,6 +46,7 @@
 #include <akonadi/agentinstance.h>
 #include <akonadi/agentinstancecreatejob.h>
 #include <akonadi/attributefactory.h>
+#include <akonadi/control.h>
 #include <akonadi/tag.h>
 #include <akonadi/tagcreatejob.h>
 #include <akonadi/tagattribute.h>
@@ -390,6 +391,10 @@
   cleanupConfigFile();
   migrateConfigurationDialogRestriction();
   deleteLater();
+  emit message( Info, i18nc("@info: progress report",
+                            "Restarting akonadi"));
+
+  Akonadi::Control::restart();
 }
 
 OrgKdeAkonadiImapSettingsInterface* KMailMigrator::createImapSettingsInterface( const Akonadi::AgentInstance& instance )
